### PR TITLE
Add Dutch (Nederlands)

### PR DESCRIPTION
**Pull request by evaeth**
Add Dutch language

### CODE REVIEW

File: svelte/src/libs/translations/index.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ import no from "./languages/no.json";
 import hr from "./languages/hr.json";
 import ja from "./languages/ja.json";
 import lt from "./languages/lt.json";
+import lt from "./languages/nl.json";
```
Comment (DavidKorystov) [inline svelte/src/libs/translations/index.ts:23]:
```suggestion
import nl from "./languages/nl.json";
```

**Review by DavidKorystov — CHANGES_REQUESTED** [review]

File: svelte/src/libs/translations/index.ts (hunk 1 above)
Comment (evaeth) [reply svelte/src/libs/translations/index.ts]: Done!

**Review by DavidKorystov — APPROVED** [review]

File: svelte/src/libs/translations/index.ts (hunk 1 above)
Comment (evaeth) [reply svelte/src/libs/translations/index.ts]: Can this PR be merged?

**Comment by getneil** [comment]
@DavidKorystov please fix conflicts thanks

**Comment by getneil** [comment]
@evaeth please fix conflicts thanks

**Comment by evaeth** [comment]
> @evaeth please fix conflicts thanks

Fixed

**Comment by evaeth** [comment]
@getneil are there any other issues?

**Comment by getneil** [comment]
@evaeth  please fix conflict again. sorry as i merge other prs from other contributors i just approve/merge those that addresses their issue the earliest.

**Comment by evaeth** [comment]
@getneil No problem. Fixed

**Comment by evaeth** [comment]
@getneil any other issues?

**Comment by getneil** [comment]
@evaeth did not pass the lint. executing `xc format` would automatically do it.

**Comment by evaeth** [comment]
@getneil any advice how to fix it?

**Comment by evaeth** [comment]
@getneil now it should pass

**Comment by evaeth** [comment]
@getneil one more try

**Comment by evaeth** [comment]
@getneil could you please check it

**Pull request merged**